doc/man1/openssl-verify.pod.in

```diff
@@ -78,8 +78,9 @@ current system time. I<timestamp> is the number of seconds since
 
 =item B<-check_ss_sig>
 
-Verify the signature on the self-signed root CA. This is disabled by default
-because it doesn't add any security.
+Verify the signature on the last certificate in a chain,
+which typically is a trusted self-signed root CA certificate.
```

Comment:
Thanks for your further comments and the explanation given here. 
Indeed my last version of this sentence was rather misleading, so I've improved it to:
```
Verify the signature on the last certificate in a chain
even when it is a self-signed (root CA) certificate.
```
which I hope is to the point and clear now, also together with the following sentence:
```
In this case the check is disabled by default
because it does not add any security.
```

Reply:
I wonder what you meant by "after checking that xi is not expired" when it is the last cert (i.e., the trust anchor) in case is not self-signed? I cannot find in `internal_verify()` such an expiration check for `xi` in this case.
If the trust anchor `xi` is not (apparently) self-signed and execution jumps into the subsequent `while` loop (which is not a recommended coding style BTW) to the label `check_cert`, the flag `X509_V_FLAG_PARTIAL_CHAIN` must have been set.

The call to `x509_check_cert_time()` only checks the expiration of `xs`, not of `xi`. Am I overlooking something here or is the expiration check on `xi` intended but missing in this case?

Reply:
Ah, I overlooked that in this case `xi = xs`, so its expiration does get checked :)